recipes/xlsxio/all/conanfile.py

```diff
@@ -0,0 +1,144 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout, CMakeDeps
+from conan.tools.env import VirtualBuildEnv
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, rmdir
+from conan.tools.scm import Version
+import os
+
+required_conan_version = ">=1.54.0"
+
+
+class XlsxioConan(ConanFile):
+    name = "xlsxio"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/brechtsanders/xlsxio"
+    description = "Cross-platform C library for reading values from and writing values to .xlsx files."
+    topics = ("xlsx",)
+    license = "MIT"
+
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "fPIC":[True, False],
+        "shared": [True, False],
+        "build_pc_files": [True, False],
+        "build_tools": [True, False],
+        "build_examples": [True, False],
+        "with_libzip": [True, False],
+        "with_minizip_ng": [True, False],
+        "with_wide": [True, False],
+        "build_documentation": [True, False],
+    }
+    default_options = {
+        "fPIC": True,
+        "shared": False,
+        "build_pc_files": True,
+        "build_tools": True,
+        "build_examples": True,
+        "with_libzip": False,
+        "with_minizip_ng": False,
+        "with_wide": False,
+        "build_documentation": False,
```

Comment:
same here with the documentation option. I think it is not needed here

Reply:
OK